.github/workflows/ci.yml

```diff
@@ -0,0 +1,93 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  CI:
+    env:
+      LOCAL_CI_TESTS_CACHEDIR: ${{ GITHUB.WORKSPACE }}/cachedir
+      LOCAL_CI_TESTS_GITDIR: "${{ GITHUB.WORKSPACE }}/moodle"
+      LOCAL_CI_TESTS_PHPCS_DIR: "${{ GITHUB.WORKSPACE }}/moodle-local_codechecker/moodle"
+      LOCAL_CI_TESTS_DBLIBRARY: native
+      LOCAL_CI_TESTS_DBTYPE: mysqli
+      LOCAL_CI_TESTS_DBHOST: 127.0.0.1
+      LOCAL_CI_TESTS_DBUSER: moodle
+      LOCAL_CI_TESTS_DBPASS: ""
+
+    services:
+      mysql:
+        image: mysql:5.7
+        env:
+          MYSQL_DATABASE: example
+          MYSQL_USER: ${{ env.LOCAL_CI_TESTS_DBUSER }}
+          MYSQL_PASSWORD: ""
```

Comment:
Hi @andrewnicols,
should it be:
```yaml
          MYSQL_PASSWORD: ${{ env.LOCAL_CI_TESTS_DBPASS}}
```
to be future-proof against any change about the idea to use empty passwords?

HTH,
Matteo